src/CodeGen/ODataT4CodeGenerator.cs

```diff
@@ -23,6 +23,7 @@ namespace Microsoft.OData.Client.Design.T4
     using Microsoft.OData.Edm.Vocabularies.V1;
     using Microsoft.OData.Edm.Vocabularies.Community.V1;
     using System.Text;
+    using System.Text.RegularExpressions;

```

Comment:
```suggestion
```
Seems like this import is no longer necessary

Reply:
Done